<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>theme_window</class>
 <widget class="QDialog" name="theme_window">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>608</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wybierz motyw</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-color:white}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="push1Theme">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>180</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#push1Theme {
background-color: transparent;
border-image:url(:/themes/themes/Flat/themeFlatNop.jpg);
background: none;
border: none;
background-repeat: none;
}
#push1Theme:checked
{
border-image:url(:/themes/themes/Flat/themeFlatHover.png);
}


</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="push3Theme">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>180</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#push3Theme {
background-color: transparent;
border-image:url(:/themes/themes/Win95/themeWin95Nop.jpg);
background: none;
border: none;
background-repeat: none;
}
#push3Theme:checked
{
border-image:url(:/themes/themes/Win95/themeWin95Hover.png);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="push2Theme">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>180</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#push2Theme {
background-color: transparent;
border-image:url(:/themes/themes/Draft/themeDraftNop.jpg);
background: none;
border: none;
background-repeat: none;
}
#push2Theme:checked
{
border-image:url(:/themes/themes/Draft/themeDraftHover.png);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="push4Theme">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>180</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#push4Theme {
background-color: transparent;
border-image:url(:/themes/themes/Default/themeDefaultNop.jpg);
background: none;
border: none;
background-repeat: none;
}
#push4Theme:checked
{
border-image:url(:/themes/themes/Default/themeDefaultHover.png);
}

</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>push4Theme</zorder>
     <zorder>push1Theme</zorder>
     <zorder>push2Theme</zorder>
     <zorder>push3Theme</zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushOkTheme">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushOkTheme {
background-color: transparent;
border-image: url(:/buttons/okOptionsNop.png);
background: none;
border: none;
background-repeat: none;
}
#pushOkTheme:pressed
{
border-image:url(:/buttons/okOptionsPressed.png);
}
#pushOkTheme:hover
{
border-image: url(:/buttons/okOptionsPressed.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushCancelTheme">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushCancelTheme {
background-color: transparent;
border-image: url(:/buttons/cancelOptionsNop.png);
background: none;
border: none;
background-repeat: none;
}
#pushCancelTheme:pressed
{
border-image:url(:/buttons/cancelOptionsPressed.png);
}
#pushCancelTheme:hover
{
border-image: url(:/buttons/cancelOptionsPressed.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
